doc/api/os.md

```diff
@@ -416,6 +416,19 @@ changes:
 Returns the operating system's default directory for temporary files as a
 string.
 
+On Windows, the result can be overridden by `TEMP` and `TMP` environment variables, and
+`TEMP` takes precedence over `TMP`. If neither is set, it defaults to `%SystemRoot%\temp`
+or `%windir%\temp`.
+
+On non-Windows platforms, `TMPDIR`, `TMP` and `TEMP` environment variables will be checked
```

Comment:
It feels strange to me that `TMP` and `TEMP` are checked in different order on Windows and non-Windows platforms. Maybe it would be better to make it consistent, but then that should be done in a different PR.